### DISCUSSION TITLE
404 on Vanjaro Api Call

### DISCUSSION THREAD

**Discussion by Scippy** [vanjarosoftware/Vanjaro.Platform#1588] [Q&A]
I've installed Vanjaro_For_DNN_1.1.0_x64_Install.zip on latest DNN release and after installed I've created a new Vanjarro page, the problem is that the page don't stop to load. Looking with web dev tools I can see that the Vanjaro Api call http://www.testsite.com/api/vanjaro/page/get?_=1628169543689 return 404
In my DNN Installation I use [Open Url Rewriter](https://github.com/sachatrauwaen/OpenUrlRewriter/releases), could be it the problem? Is there any workaround to fix it?

**Comment by mksmanish79** [comment]
http://www.testsite.com/api/vanjaro/page/get?_=1628169543689

Is domain url correct? Same as the tab url of your browser.

**Reply by Scippy** [reply]
It is localhost enviroments

**Reply by mksmanish79** [reply]
Please check your portal alias table and make sure you have only 1 primary site for your portal you are facing the issue on. Check Isprimary column and make sure only 1 entry is primay for the portal you are facing the issue.

**Reply by Scippy** [reply]
Yes, I've only one portal alias defined for this website and it is primary

**Reply by mksmanish79** [reply]
I am able to duplicate the issue with Open Url Rewriter.

**Reply by mksmanish79** [reply]
Please contact Open Url Rewriter so that they can provide you a quick solution.
